backend/src/main/java/reviewme/review/service/CreateCheckBoxAnswerRequestValidator.java

```diff
@@ -0,0 +1,83 @@
+package reviewme.review.service;
+
+import java.util.HashSet;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import reviewme.question.domain.OptionGroup;
+import reviewme.question.domain.OptionItem;
+import reviewme.question.domain.Question2;
+import reviewme.question.repository.Question2Repository;
+import reviewme.review.dto.request.create.CreateReviewAnswerRequest;
+import reviewme.review.service.exception.CheckBoxAnswerIncludedNotProvidedOptionItemException;
+import reviewme.review.service.exception.CheckBoxAnswerIncludedTextException;
+import reviewme.review.service.exception.RequiredQuestionMustBeAnsweredException;
+import reviewme.review.service.exception.SelectedCheckBoxAnswerCountOutOfRange;
+import reviewme.template.domain.exception.OptionGroupNotFoundException;
+import reviewme.template.repository.OptionGroupRepository;
+import reviewme.template.repository.OptionItemRepository;
+
+@Service
+@RequiredArgsConstructor
+public class CreateCheckBoxAnswerRequestValidator {
+
+    private final Question2Repository question2Repository;
+    private final OptionGroupRepository optionGroupRepository;
+    private final OptionItemRepository optionItemRepository;
+
+    public void validate(CreateReviewAnswerRequest request) {
+        validateNotContainingText(request);
+        Question2 question = validateQuestionExists(request);
+        OptionGroup optionGroup = validateOptionGroupExists(question);
```

Comment:
validate 메서드가 리턴값이 있는게 이해하는데 혼선을 줄 수 있다고 생각해요!
처음부터 변수로 빼는 건 어떨까요?
```suggestion
        OptionGroup optionGroup = optionGroupRepository.findByQuestionId(question.getId())
                .orElseThrow(() -> new OptionGroupNotFoundException(question.getId()));
        Question2 question = question2Repository.getQuestionById(questionId)
```

Reply:
사실 저도 validate 함수가 리턴을 하게 할까, 말까 고민이 되었는데,
validate 가 촤르륵 적혀있으면 각 줄에 뭘 검증하는지 보기가 편해서 저렇게 해뒀었습니다 ㅎㅎ
그런데 혼선을 줄 수 있다면 커비가 제안하는 방향으로 변경하겠습니다!